tests/test_indexer.py

```diff
@@ -382,6 +383,20 @@ def get_search_results(cls, query, expected_hit_count):
             else:
                 time.sleep(0.5)
 
+    def clear_test_indexes(self):
+        try:
+            es_client = ElasticsearchClient.get(logger)
+            es_client.delete_by_query(index=self.dss_index_name,
+                                      body={'query': {'match_all': {}}},
+                                      doc_type=[dss.ESDocType.doc.name, dss.ESDocType.query.name],
+                                      refresh=True)
+            es_client.delete_by_query(index=self.subscription_index_name,
+                                      body={'query': {'match_all': {}}},
+                                      doc_type=dss.ESDocType.subscription.name,
+                                      refresh=True)
+        except TransportError:
```

Comment:
Under what circumstances would a `TransportError` occur?  If it happens, it is better that we just crash and burn?

Reply:
This occurs when the index has not been created and I try to delete from it. I could narrow the used of it to the specific test cases where the index is not created yet.